### PR TITLE
fix: fix issues with scripts

### PR DESCRIPTION
**Pull request by nnyyxxxx**
## Type of Change
- [x] Bug fix

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -111,26 +111,33 @@ installDeps() {
 setupConfigurations() {
     printf "%b\n" "${YELLOW}Setting up configuration files...${RC}"
 
+    mkdir -p "$XDG_CONFIG_HOME/cava"
+    mkdir -p "$XDG_CONFIG_HOME/fastfetch"
+    mkdir -p "$XDG_CONFIG_HOME/nvim"
+    mkdir -p "$XDG_CONFIG_HOME/gtk-3.0"
+    mkdir -p "$XDG_CONFIG_HOME/picom"
+    mkdir -p "$XDG_CONFIG_HOME/qt5ct"
+
```
Comment (adamperkowski) [inline install.sh:120]: no need for this. `ln -sf`s create directories

**Review by adamperkowski — CHANGES_REQUESTED** [review]

File: install.sh (hunk 1 above)
Comment (nnyyxxxx) [reply install.sh:120]: wrong

Comment (adamperkowski) [reply install.sh:120]: nope. this links the dirs, not files
https://github.com/nnyyxxxx/dwm/blob/7a23a74e31d0db49c4d0eef6295ae67e4f58b14c/install.sh#L123-L131

Comment (adamperkowski) [reply install.sh:120]: on a fresh install

image

Comment (nnyyxxxx) [reply install.sh:120]: > nope. this links the dirs, not files
> 
> https://github.com/nnyyxxxx/dwm/blob/7a23a74e31d0db49c4d0eef6295ae67e4f58b14c/install.sh#L123-L131

ah ok

**Review by adamperkowski — APPROVED** [review]

**Pull request merged**